opentelemetry-zipkin/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## vNext
 
+### Changed
+
+- Update to `http` v1 [#1674](https://github.com/open-telemetry/opentelemetry-rust/pull/1674)
```

Comment:
Not sure this is the best way to explain this change? The material change for the -zipkin API seems to be that it upgraded opentelemetry-http, from which it uses the `HyperClient` trait.

Reply:
Yeah, I was absent-mindedly adding that line to changelogs where I changed something re http/hyper v1.  
This is more of an "update examples to hyper v1" situation. Adjusted.